From: Dinar Valeev <dvaleev@suse.com>
Date: Thu, 12 Sep 2013 12:51:31 +0200

build: resolve compile abort on big-endian platforms

s was not declared.

Addition by jengelh: Make it unsigned, because that is the
only thing to make sense with the >> operator.

Index: SDL2_image-2.0.0/IMG_webp.c
===================================================================
--- SDL2_image-2.0.0.orig/IMG_webp.c
+++ SDL2_image-2.0.0/IMG_webp.c
@@ -242,6 +242,7 @@ SDL_Surface *IMG_LoadWEBP_RW(SDL_RWops *
     Bmask = 0x00FF0000;
     Amask = (features.has_alpha) ? 0xFF000000 : 0;
 #else
+    unsigned int s;
     s = (features.has_alpha) ? 0 : 8;
     Rmask = 0xFF000000 >> s;
     Gmask = 0x00FF0000 >> s;
